Source/WebCore/html/HTMLEmbedElement.cpp

```diff
@@ -82,10 +82,8 @@ RenderWidget* HTMLEmbedElement::renderWidgetLoadingPlugin() const
 void HTMLEmbedElement::collectPresentationalHintsForAttribute(const QualifiedName& name, const AtomString& value, MutableStyleProperties& style)
 {
     if (name == hiddenAttr) {
-        if (equalLettersIgnoringASCIICase(value, "yes"_s) || equalLettersIgnoringASCIICase(value, "true"_s)) {
-            addPropertyToPresentationalHintStyle(style, CSSPropertyWidth, 0, CSSUnitType::CSS_PX);
-            addPropertyToPresentationalHintStyle(style, CSSPropertyHeight, 0, CSSUnitType::CSS_PX);
-        }
+        addPropertyToPresentationalHintStyle(style, CSSPropertyWidth, 0, CSSUnitType::CSS_PX);
+        addPropertyToPresentationalHintStyle(style, CSSPropertyHeight, 0, CSSUnitType::CSS_PX);
```

Comment:
```suggestion
        ASSERT(!value.isNull());
        addPropertyToPresentationalHintStyle(style, CSSPropertyWidth, 0, CSSUnitType::CSS_PX);
        addPropertyToPresentationalHintStyle(style, CSSPropertyHeight, 0, CSSUnitType::CSS_PX);
```
I wonder if we should assert this, it would be wrong to apply this if the value is null (e.g. no hidden attribute present).

@rniwa Wdyt?

Reply:
Good point IMO!. Will wait for @rniwa input but happy to do change.

Reply:
Yeah, asserting that makes sene to me.

Reply:
Done.